presto-cassandra/src/test/java/com/facebook/presto/cassandra/CassandraTestingUtils.java

```diff
@@ -148,6 +150,17 @@ public static void createTableAllTypesPartitionKey(Session session, String keysp
         insertTestData(session, keyspace, tableName, date);
     }
 
+    public static void createTableWithEmptyBlobCell(Session session, String keyspace, String tableName, Date date)
+    {
+        session.execute("DROP TABLE IF EXISTS " + tableName);
```

Comment:
Put this inside the test method in `TestCassandraIntegrationSmokeTest`. Splitting this over two classes makes it a lot harder to read.